rust/crd/src/lib.rs

```diff
@@ -22,26 +20,30 @@ use stackable_operator::{
     role_utils::{GenericRoleConfig, Role, RoleGroup, RoleGroupRef},
     schemars::{self, JsonSchema},
     status::condition::{ClusterCondition, HasStatusCondition},
+    time::Duration,
 };
+use std::collections::BTreeMap;
 use strum::{Display, EnumIter, EnumString, IntoEnumIterator};
 
-use crate::{affinity::get_affinity, authentication::SupersetAuthentication};
-
 pub mod affinity;
 pub mod authentication;
 pub mod druidconnection;
 
+use crate::{affinity::get_affinity, authentication::SupersetAuthentication};
```

Comment:
Module definitions should be placed after all imports.

Reply:
https://github.com/stackabletech/superset-operator/pull/422/commits/6889c0c82d519d2896d3a346ccca117496bbb83e